openpype/hosts/houdini/plugins/publish/collect_instances.py

```diff
@@ -102,4 +102,4 @@ def get_frame_data(self, node):
 
         """
 
-        return lib.get_frame_data(self, node)
+        return lib.get_frame_data(node)
```

Comment:
In this case I'd argue to just remove the `get_frame_data` on the plugin and refactor it so it directly uses `lib.get_frame_data` where it's used. Then I noticed that the `get_frame_data` method here isn't used at all haha - so that can be removed anyway.

And actually, CollectInstances I believe is a Collector for Legacy instances (not the new style instances) so it seems this whole plugin could be removed to begin with anyways? but that's likely best left to another PR)

For now I'd say just remove the whole method definition since it seem unused.